sw/airborne/arch/chibios/STM32F76xxI.ld

```diff
@@ -0,0 +1,132 @@
+/*
+    ChibiOS - Copyright (C) 2006..2018 Giovanni Di Sirio
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+/*
+ * STM32F76xxI generic setup.

```

Comment:
Why is this file needed here ? It should be in `sw/ext/chibios/os/common/startup/ARMCMx/compilers/GCC/ld/` already, we are using it for Tawaki board.

Reply:
I think it is because I shifted the flash 0x08008000 to keep the bootloader

Reply:
right, I was wondering where this is selected, but it is in chibios makefile in conf, so it's fine